inst/extremes/extremes.Rmd

```diff
@@ -26,5 +26,8 @@ cat(getLogo())
 tbl <- extremesTable(data)
 # formTable <- padTable(tbl)
 # cat(formTable)
-kable(tbl, align='r', row.names=FALSE)
+if(!identical(tbl, "The dataset requested is empty.")){
```

Comment:
this logic is all over repgen.  I can change it here, but we should put in a tech debt task to apply the change everywhere.